README.md

```diff
@@ -0,0 +1,24 @@
+# SQurreL SQL - Flatpak
```

Comment:
In general there's no need to have README as it's unlikely any user will read it.

Reply:
I removed the README.